### ISSUE TITLE
Neuromancer KAN Example 1 imports LossLogger from neuromancer.logging but LossLogger does  not exist

### ISSUE THREAD

**Issue by ejoly3** [pnnl/neuromancer#196]
Part of the example shown here: https://github.com/pnnl/neuromancer/blob/feature/fbkans/examples/KANs/p1_fbkan_vs_kan_noise_data_1d.ipynb
attempts to import LossLogger from the file neuromancer/src/neuromancer/loggers.py
However, LossLogger is not an object in the file loggers.py

**Comment by brunopjacob** [comment]
Hi @ejoly3! Thanks for using our library!

I just double-checked and it seems that the LossLogger [is there](https://github.com/pnnl/neuromancer/blob/224acc8617b7c706fc4d986361334d58bee5babd/src/neuromancer/loggers.py#L88-L130), but perhaps your Neuromancer installation is still pointing to the `develop` or `master`? Note that the fbkan work is still waiting to be merged (it will be merged soon! Perhaps today!), so installing it via the pypi wheel won't allow you to use this feature until we merge it to master, but in the meantime you can install it locally as explained below.

Please give the following a try: 

1) Clone neuromancer on the feature/fbkans branch:
```
git clone --branch feature/fbkans https://github.com/pnnl/neuromancer.git
```

2) Enter the cloned repository, and install it locally (we recommend you use some sort of virtual environment here!)
cd neuromancer
python3 -m venv .venv
source .venv/bin/activate
pip install -e .[docs,tests,examples]
```

3) Give that notebook a try. You might still need to install Jupyter, depending on your setup.

Please let us know if that fixed the issue! I'll be happy to help.

Thanks!
Bruno

**Comment by ejoly3** [comment]
Yes, the issue was it is looking in the master directory. Thanks for the fix!
